methods.go

```diff
@@ -139,6 +155,22 @@ func (c *Client) TorrentsStatus() (map[string]*TorrentStatus, error) {
 	return decodeTorrentsStatusResponse(resp)
 }
 
+func decodeTorrentStatusResponse(resp *DelugeResponse) (*TorrentStatus, error) {
+	values := resp.returnValue.Values()
+	if len(values) != 1 {
+		return nil, ErrInvalidReturnValue
+	}
+	rd, ok := values[0].(rencode.Dictionary)
+	if !ok {
+		return nil, ErrInvalidListResult
+	}
+
+	var ts TorrentStatus
+	err := rd.ToStruct(&ts)
+
```

Comment:
It would be best for consistency to return `nil` in case of error here.

Reply:
Error is no longer returned in latest version

Reply:
We should not ignore errors here; back then I assumed that you meant that `ToStruct` does not return errors anymore, but it does.

This issue has been fixed in 773914eab268cd89b614c7c123fb84c1f7d1fdae